staking-miner-playground/runtime/src/lib.rs

```diff
@@ -155,34 +153,25 @@ parameter_types! {
 	pub const BlockHashCount: BlockNumber = 2400;
 	pub const SS58Prefix: u8 = 42;
 
-	pub ConfigurableBlockLength: limits::BlockLength = limits::BlockLength { max: PerDispatchClass::new(|_| ConfigBlock::block_length()) };
-	pub ConfigurableBlockWeight: limits::BlockWeights = limits::BlockWeights::simple_max(Weight::from_parts(ConfigBlock::block_weight(), u64::MAX));
```

Comment:
This is removed because the `configurable block weight/length` doesn't work when running it with subxt because it relies on the metadata and this has no effect.

The only way is to change block length and block weight is by runtime upgrade.